include/pops/date.hpp

```diff
@@ -443,6 +444,29 @@ int Date::weeks_from_date(Date start)
     return week - 1;
 }
 
+/*!
+ * Returns the current date as a string
+ * \param Date, current date
+ * \return String, current date as string
+ */
+std::string Date::to_string()
+{
+    std::string date = std::to_string(year_) + "-";
+    if (month_ < 10) {
+        date = date + "0" + std::to_string(month_) + "-";
+    }
+    else {
+        date = date + std::to_string(month_) + "-";
+    }
+    if (day_ < 10) {
+        date = date + "0" + std::to_string(day_);
+    }
+    else {
+        date = date + std::to_string(day_);
```

Comment:
The usual style would be to use `date += std::to_string(day_);` here which documents the intention (adding to date as opposed to replacing value of date).